utkarsh component/modal/modal.css

```diff
@@ -0,0 +1,410 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+Thai+Looped:wght@300&display=swap');
+body{
+    font-family: 'IBM Plex Sans Thai Looped', sans-serif;
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+background-color:  #31353a;
+height: 100%;
+
+}
+
+* {
+   box-sizing: border-box;
+--primary-color: #ecfeff ;
+--secondary-color: #4908fa;
+}
+
+.container{
+    color: var(--primary-color);
+    width: 100vw;
+    text-decoration: none;
+    display: grid;
+    font-weight: 400;
+    grid-auto-flow: column;
+    grid-template-columns: 200px 1fr; 
+ 
+    grid-template-areas: 
+    "nav nav "
+    "pitch pitch"
+   "aside main"
+   "aside main"
+   "aside main"
+   "footer footer"
+}
+
+
+.imp-text{
+    font-family: 'Redressed', cursive;
+font-family: 'Roboto', sans-serif;
+font-family: 'Satisfy', cursive;
+color: var(--secondary-color);
+text-align: left;
+
+}
+
+
+
+nav ul{
+    margin-left: auto; 
+    display: flex;
+  gap: 15px;
+  position: relative;
+  bottom: 35px;
+  padding-inline-end: 0.7rem;
+ 
+}
+
+li{
+ 
+    list-style: none;
+    padding: 0.5rem;
+    font-weight: bold;
+   
+}
+
+
+main{
+    
+  /* grid-template-areas: 200px 1fr; */
+  /* align-items: center; 
+  padding: 3.5rem;
+  text-align: center;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  margin: auto; */
+  grid-area: main;
+  font-size: 1.2rem;
+  padding-left: 7rem;
+  padding-right: 7rem;
+  
+ 
+}
+
+main p{
+  letter-spacing: 0.1rem;
+}
+h3{
+  font-size: 1.5rem;
+  /* line-height: 2rem; */
+  letter-spacing: 0.1rem;
+}
+
+.impact{
+   font-style: italic;
+   color: var(--secondary-color);
+   
+}
+
+ 
+ 
+
+ .component-text{
+  grid-area: pitch;
+  font-size: 42.5%;
+  font-size: 2rem; 
+  margin: auto;
+  letter-spacing: 0.1rem;
+}
+
+
+hr{
+  border-top: 1px solid var(--primary-color); 
+width: 100rem;
+}
+
+.highlight{
+  color: hotpink;
+}
+
+iframe{
+  padding-top: 1rem;
+}
+
+
+
+
+
+.medium{
+    width: 4.5rem;
+    height: 4.5rem;
+    margin-left: 2rem; 
+}
+
+.large{
+    width: 5.5rem;
+    height: 5.5rem;
+    margin-left: 2rem; 
+}
+
+
+
+
+footer{
+  grid-area: footer;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 0.5rem;
+  color: #fff;
+    
+
+}
+
+footer ul{
+padding-inline-start: 0.5rem;
+  display: flex;
+gap: 20px;
+color: #fff;

```

Comment:
use var (variable) for color and avoid using the color code, it will your code more readable and also easier to alter